### DISCUSSION TITLE
A way to assign multiple actions to a mouse button

### DISCUSSION THREAD

**Discussion by dudethis** [sezanzeb/input-remapper#108] [Ideas]
hey It would be nice to be able to assign multiple clicks for every click I do on my mouse so I would be able to double click in minecraft (if that makes any sense)  thanks for all of the effort your putting into this project. (also im new to github so sorry if i put this in the wrong place)

**Comment by sezanzeb** [comment]
`k(BTN_LEFT).k(BTN_LEFT)`

you might have to add a small pause between the two clicks, like:

`k(BTN_LEFT).w(50).k(BTN_LEFT)`

or, equivalent, use the "repeat" function:

`r(2, k(BTN_LEFT).w(50))`

see https://github.com/sezanzeb/key-mapper/blob/main/readme/macros.md
